woocommerce/woocommerce-android
# Add toolbar to PrintShippingLabelFragment

Part of #10570 

### Description

This PR introduces individual toolbars for PrintShippingLabelFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Order detail screen of an Order that has a shipping label (PbTz5e-11v-p2) (Order list -> Order detail -> Print shipping label) screen and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif

<img width="243" alt="Screenshot 2024-02-12 at 6 25 22 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/aeb0fad6-444d-4cd0-8656-a242f8689aaa">
